motrainer/jackknife.py

```diff
@@ -223,7 +223,9 @@ def export_best(self, model_name='best_optimized_model',
         metedata['input_list'] = self.input_list
         metedata['output_list'] = self.input_list
         metedata['best_year'] = int(self.best_year)
-        metedata['lat'] = float(self.gpi_data['lat'].iloc[0])
-        metedata['lon'] = float(self.gpi_data['lon'].iloc[0])
+        if 'lat' in self.gpi_data.keys():
+            metedata['lat'] = float(self.gpi_data['lat'].iloc[0])
+        if 'lon' in self.gpi_data.keys():
+            metedata['lon'] = float(self.gpi_data['lon'].iloc[0])
```

Comment:
I like that you implemented the check for coordinates. This part of the code is also changed in #97 with respects to the coordinate names that should be `latitude` and `longitude`. can you please merge the `generalization` into your branch and resolve the merge conflicts?